### PR TITLE
testing/i2pd: upgrade to 2.25.0

### PR DESCRIPTION
**Pull request by r4sas**

### CODE REVIEW

File: testing/i2pd/APKBUILD [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributor: l-n-s <supervillain@riseup.net>
 # Maintainer: l-n-s <supervillain@riseup.net>
 pkgname=i2pd
-pkgver=2.24.0
-pkgrel=1
+pkgver=2.25.0
+pkgrel=0
 pkgdesc="I2P Router written in C++"
 url="https://github.com/PurpleI2P/i2pd"
 arch="all"
```
Comment (maxice8) [inline testing/i2pd/APKBUILD]: license is not SPDX

Comment (maxice8) [reply testing/i2pd/APKBUILD]: if you're adding openssl-dev then it brings openssl, so the openssl dependency is redundant

Comment (maxice8) [reply testing/i2pd/APKBUILD]: has conf.d and init.d files but no $pkgname-openrc

Comment (maxice8) [reply testing/i2pd/APKBUILD]: would be nice to have a comment on why check is disabled

Comment (maxice8) [reply testing/i2pd/APKBUILD]: bulddir= is using default value so it can be removed

Comment (maxice8) [reply testing/i2pd/APKBUILD]: no need for `cd "$builddir"`

**Comment by r4sas** [comment]
Oops, wrong button. 

About license - change to identifier from SPDX list?

**Comment by maxice8** [comment]
https://wiki.alpinelinux.org/wiki/Creating_an_Alpine_package#license

**Comment by r4sas** [comment]
> has conf.d and init.d files but no $pkgname-openrc

Still dont understand about that, because here already present `subpackages="$pkgname-openrc"`.

**Comment by maxice8** [comment]
yes, i missed it.

**Comment by andypost** [comment]
please squash commits into single one

**Pull request closed without merging**